src/components/ListDetail/ListDetail.tsx

```diff
@@ -295,9 +298,21 @@ export const ListDetail = ({
                         purpose="decorative"
                       />
                     ) : itemVariant === 'number' ? (
-                      <span className={styles['list-detail__number']}>
-                        {i + 1}
-                      </span>
+                      <span className={styles['list-detail__number']}>{i}</span>
+                    ) : itemVariant === 'incomplete' ? (
+                      <Icon
+                        className={styles['list-detail__icon']}
+                        color={EdsThemeColorBorderNeutralSubtle}
+                        name="star" /* TODO: need to add star-outline variant to Icon */
+                        purpose="decorative"
```

Comment:
Should these icons instead be "informative" and have a `title`? Are the states they communicate also stated in text somewhere so a screen reader user knows that there's an error or warning or it's successfully completed, etc?

Reply:
Sounds good, thanks! 👍